server/cmwell-ws/app/logic/CRUDServiceFS.scala

```diff
@@ -800,7 +800,7 @@ class CRUDServiceFS @Inject()(implicit ec: ExecutionContext, sys: ActorSystem) e
 
   private def toFieldValues(ss: Set[String]): Set[FieldValue] = ss.map(FString.apply)
 
-  def startScroll(pathFilter: Option[PathFilter] = None,
+  def startScrollEliNew(pathFilter: Option[PathFilter] = None,
```

Comment:
please `sed s/EliNew//g` all over the place... :)

Reply:
done